app/controllers/dashboard_controller.rb

```diff
@@ -2,6 +2,6 @@ class DashboardController < ApplicationController
   before_action :authorize!
 
   def index
-    @user = current_user
+    @user = ViewUser.build(current_user.token, current_user.uid)
```

Comment:
I probably wouldn't change it at this point, but I'm wondering if I would like `Redditor` better than ViewUser. I think this is fine, and I'm definitely being picky. Naming things is hard.